### PR TITLE
MzML bug fixes and addition of precursor scan number to isolation window

### PR DESCRIPTION
**Pull request by adhi1419**
--Bug Fixes
-Fix position of `IsolationWindow`
-Change size of float encoded `ByteBuffer`
-Parse Default Instrumentation Configuration from `run` tag
-Parse Default Data Processing from `spectrum` and `chromatogtam` tags
-Use mzXML file with polarity attribute for JUnit test cases

--Isolation Window
-Add precursor scan number field

### CODE REVIEW

**Comment by tomas-pluskal** [comment]
It seems the Travis build failed because of missing module msdk-io-mz5, did you add that module?

File: pom.xml [hunk 1]
```diff
@@ -29,6 +29,7 @@
 		<module>msdk-id-metfrag</module>
 		<module>msdk-id-sirius</module>
 		<module>msdk-io-filetypedetection</module>
+		<module>msdk-io-mz5</module>
```
Comment (tomas-pluskal) [inline pom.xml]: Perhaps remove this?

Comment (adhi1419) [reply pom.xml]: Sorry, somehow progress from my other branch stayed to this while switching branches. I removed the module folder from this branch, but, forgot to remove it from pom.xml

**Comment by tomas-pluskal** [comment]
Thanks! I'll merge this now, but could you take at the failing tests?

**Pull request merged**